packages/code-du-travail-frontend/cypress/integration/light/politique-confidentialite.spec.ts

```diff
@@ -0,0 +1,10 @@
+describe("Pages informations", () => {
+  it("je vois une page info classique", () => {
+    cy.visit("/politique-confidentialite");
+    cy.get("h1").should("have.text", "Politique de confidentialité");
```

Comment:
ça ne doit pas marcher ici car il trouve deux `h1`.

```suggestion
    cy.contains("h1",  "Politique de confidentialité");
```

Reply:
Après on pourra passer comme ce qu'a fait @carolineBda sur sa PR avec la lib testing-library

Reply:
Pas sûr que ça marche du coup ce qu'elle a mis en place @carolineBda car on a deux `h1` 🤔